app/serializers/organization_serializer.rb

```diff
@@ -0,0 +1,32 @@
+class OrganizationSerializer
+  include RestPack::Serializer
+
+  attributes :id, :name, :display_name, :description, :introduction, :title, :href
+
+ def title
+   content[:title]
+ end
+
+ def description
+   content[:description]
+ end
+
+ def introduction
+   content[:introduction]
+ end
+
+ def content
+   @content ||= _content
+ end
+
+ def fields
+   %i(title description introduction)
+ end
+
+ def _content
```

Comment:
✔️ , noticed that I didn't add a spec for the org serializer, but I'm not sure that's necessary as it just now doesn't include anything unexpected.